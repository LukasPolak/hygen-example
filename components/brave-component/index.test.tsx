import { render } from '@testing-library/react'

import { BraveComponent } from '.'

describe('BraveComponent', () => {
  const component = <BraveComponent>BraveComponent</BraveComponent>

  it('should render without crashing', () => {
    render(component)
  })

  it('should match previous snapshots', () => {
    const { baseElement } = render(component)
    expect(baseElement).toMatchSnapshot('base')
  })
})
