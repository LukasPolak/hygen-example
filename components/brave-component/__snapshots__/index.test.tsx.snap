// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BraveComponent should match previous snapshots: base 1`] = `
<body>
  <div>
    <div
      class="sc-bdfBwQ"
    >
      BraveComponent
    </div>
  </div>
</body>
`;
